resources.html

```diff
@@ -27,6 +27,25 @@ <h3>Community</h3>
         <p>
             There, you can share tips, ask questions and get acquainted with other creators.
         </p>
+
+        <h3>Setup</h3>
+
+        <p>
+            Not all programming languages are suitable for creating 64k intros. It's important to check that you can create small
+            executable files, and you'll often need to adjust the compiler options. The most common languages are C++, Rust, and
+            Javascript.
+        </p>
+        <p>
+            These links can help you get started:
+        </p>
+        <ul>
+            <li><a href="https://github.com/Kwarf/64k-starter">64k-starter</a> is a good starting point for writing a 64k intro in
+                Rust.
+            <li><a href="https://madethisthing.com/iq/Demo-Framework-64k">Demo-Framework-64k</a> can be used a template for intros
+                in C++.
+            <li><a href="https://github.com/armak/Leviathan-2.0/">Leviathan 2.0</a> is a minimalist setup targeting intros from 1 to 8kB
+                using raymarching.
```

Comment:
Has there been 64k done with Leviathan? That sounds off topic.

Reply:
removed